pennylane/ops/__init__.py

```diff
@@ -83,5 +83,5 @@ def diagonalizing_gates(self):
 
 
 __all__ = _cv__all__ + _qubit__all__ + _channel__ops__ + ["Identity"]
-__all_ops__ = list(_cv__ops__ | _qubit__ops__)
+__all_ops__ = list(_cv__ops__ | _qubit__ops__) + ["Identity"]  # we need this as well now correct?
```

Comment:
We also need to add it here correct?

Reply:
This is mostly used for docs rendering (+imports). Do the docs look good now for this module?

One place to add identity would be here:

https://pennylane.readthedocs.io/en/stable/introduction/operations.html#non-parametric-ops

It's the `operations.rst` file in the docs.

Reply:
Added it to the `operations.rst` file, docs look good